### PR TITLE
shelter: add rpm and deb spec for package build

### PR DESCRIPTION
**Pull request by zhiminghufighting**
Fixes: #502
Signed-off-by: zhiminghufighting <zhiming.hu@intel.com>

### CODE REVIEW

File: shelter/dist/rpm/shelter.spec [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+%files
+%{BIN_DIR}/%{name}
+
+%changelog
+* Thu Apr 15 2021 Zhiming Hu <zhiming.hu@intel.com> - 0.6.1
+- Update to 0.6.1 and enable dacp remote attestation based on enclave-tls lib.
+
+* Thu Feb 07 2021 Zhiming Hu <zhiming.hu@intel.com> - 0.6.0
+- Update to 0.6.0 and enable dacp remote attestation based on wolsf-ssl lib.
+
```
Comment (haosanzi) [inline shelter/dist/rpm/shelter.spec:59]: Please fix typo `wolsf-ssl `

File: shelter/dist/deb/debian/changelog [hunk 2]
```diff
@@ -0,0 +1,17 @@
+shelter (0.6.1) unstable; urgency=low
+
```
Comment (haosanzi) [inline shelter/dist/deb/debian/changelog:2]: Change to `0.6.1-1`

File: shelter/dist/rpm/shelter.spec [hunk 3]
```diff
@@ -0,0 +1,61 @@
+%define centos_base_release 1
+%define _debugsource_template %{nil}
+
+%global PROJECT inclavare-containers
+%global BIN_DIR /usr/local/bin
+
+Name: shelter
+Version: 0.6.0
```
Comment (haosanzi) [inline shelter/dist/rpm/shelter.spec:8]: Please change to `0.6.1`

File: shelter/dist/rpm/shelter.spec (hunk 1 above)
Comment (zhiminghufighting) [reply shelter/dist/rpm/shelter.spec:59]: done

File: shelter/dist/deb/debian/changelog (hunk 2 above)
Comment (zhiminghufighting) [reply shelter/dist/deb/debian/changelog:2]: done

File: shelter/dist/rpm/shelter.spec (hunk 3 above)
Comment (zhiminghufighting) [reply shelter/dist/rpm/shelter.spec]: done

File: shelter/dist/rpm/shelter.spec [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+Name: shelter
+Version: 0.6.1
+Release: %{centos_base_release}%{?dist}
+Summary: shelter is designed as a remote attestation tool for customer to verify if their workloads are loaded in a specified intel authorized sgx enclaved.
+
+Group: Development/Tools
+License: Apache License 2.0
+URL: https://github.com/alibaba/%{PROJECT}
+Source0: https://github.com/alibaba/%{PROJECT}/archive/v%{version}.tar.gz
+
```
Comment (hustliyilin) [inline shelter/dist/rpm/shelter.spec:16]: add `BuildRequires: openssl-devel` please.

File: shelter/dist/deb/debian/control [hunk 5]
```diff
@@ -0,0 +1,12 @@
+Source: shelter
+Section: devel
+Priority: extra
+Maintainer: zhiming hu <zhiming.hu@intel.com>
+Build-Depends: debhelper (>=9)
```
Comment (hustliyilin) [inline shelter/dist/deb/debian/control:5]: add Build-Depends: openssl-dev

**Review by hustliyilin — COMMENTED** [review]
BuildRequires: enclave-tls, So can't be merged before enclave-tls rpm and deb ready

File: shelter/dist/deb/debian/changelog [hunk 6]
```diff
@@ -0,0 +1,17 @@
+shelter (0.6.1-1) unstable; urgency=low
```
Comment (hustliyilin) [inline shelter/dist/deb/debian/changelog:1]: Merge all changelogs into one please.

File: shelter/dist/rpm/shelter.spec [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+popd
+
+%install
+install -d -p %{buildroot}%{BIN_DIR}
+install -p -m 755 %{name}/%{name} %{buildroot}%{BIN_DIR}
+
+%files
+%{BIN_DIR}/%{name}
+
+%changelog
```
Comment (hustliyilin) [inline shelter/dist/rpm/shelter.spec:53]: Merge all changelogs into one please.

File: shelter/dist/rpm/shelter.spec (hunk 4 above)
Comment (zhiminghufighting) [reply shelter/dist/rpm/shelter.spec:16]: done

File: shelter/dist/deb/debian/control (hunk 5 above)
Comment (zhiminghufighting) [reply shelter/dist/deb/debian/control:5]: done

File: shelter/dist/deb/debian/changelog (hunk 6 above)
Comment (zhiminghufighting) [reply shelter/dist/deb/debian/changelog:1]: done

File: shelter/dist/rpm/shelter.spec (hunk 7 above)
Comment (zhiminghufighting) [reply shelter/dist/rpm/shelter.spec:54]: done

**Review by jiazhang0 — APPROVED** [review]

**Review by haosanzi — APPROVED** [review]

**Pull request merged**